code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -3,67 +3,67 @@
 ///////////////////////////////////
 
 /datum/design/milk
-	name = "Milk"
+	name = "Synthetic Milk"
 	id = "milk"
 	build_type = BIOGENERATOR
 	materials = list(/datum/material/biomass = 0.4)
-	make_reagent = /datum/reagent/consumable/milk
+	make_reagent = /datum/reagent/consumable/milk/synthetic
 	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_BIO_FOOD)
 
 /datum/design/soymilk
-	name = "Soy Milk"
+	name = "Synthetic Soy Milk"
 	id = "soymilk"
 	build_type = BIOGENERATOR
 	materials = list(/datum/material/biomass = 0.4)
-	make_reagent = /datum/reagent/consumable/soymilk
+	make_reagent = /datum/reagent/consumable/soymilk/synthetic
 	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_BIO_FOOD)
 
 /datum/design/ethanol
-	name = "Ethanol"
+	name = "Synthetic Ethanol"
 	id = "ethanol"
 	build_type = BIOGENERATOR
 	materials = list(/datum/material/biomass = 0.6)
-	make_reagent = /datum/reagent/consumable/ethanol
+	make_reagent = /datum/reagent/consumable/ethanol/synthetic
 	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_BIO_FOOD)
 
 /datum/design/cream
-	name = "Cream"
+	name = "Synthetic Cream"
 	id = "cream"
 	build_type = BIOGENERATOR
 	materials = list(/datum/material/biomass = 0.6)
-	make_reagent = /datum/reagent/consumable/cream
+	make_reagent = /datum/reagent/consumable/cream/synthetic
 	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_BIO_FOOD)
 
 /datum/design/black_pepper
-	name = "Black Pepper"
+	name = "Synthetic Black Pepper"
 	id = "black_pepper"
 	build_type = BIOGENERATOR
 	materials = list(/datum/material/biomass = 0.6)
-	make_reagent = /datum/reagent/consumable/blackpepper
+	make_reagent = /datum/reagent/consumable/blackpepper/synthetic
 	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_BIO_FOOD)
 
 /datum/design/enzyme
-	name = "Universal Enzyme"
+	name = "Synthetic Universal Enzyme"
```

Comment:
Synthetic universal enzyme seems a little weird, considering how universal enzyme in itself is already likely synthetic, plus considering how it's not consumed in recipes this'll likely just make this recipe completely useless.

Reply:
It is a protein, it does fill you up (although just 1 nutriment factor), you can make a pure version of it by fermenting tomatoes and the purity of catalysts will affect the purity of the result product. Since 100% pure milk is easily obtained from goat, you would make finest cheese roundstart if this wasn't considered. I may rename it to "Synthetic Enzyme" though.

Reply:
Might wanna change the bottle of enzyme in the chef's kitchen to this, then.

Reply:
Mapspawn things are better than biogen stuff on purpose.